packages/shared/src/components/squads/Details.tsx

```diff
@@ -170,6 +171,28 @@ export function SquadDetails({
               }}
             />
           )}
+          <div className="flex flex-col justify-start">
+            <h4 className="mb-2 font-bold typo-headline text-theme-label-primary">
+              Post permissions
+            </h4>
+            <p className="mb-6 text-salt-90 typo-callout">
```

Comment:
Ah we shouldn't use color versions directly, did linting not complain about it?
And margin-bottom shows as 16 to the box so 4.

```suggestion
            <p className="mb-4 text-theme-label-tertiary typo-callout">
```

Reply:
Nope, I did see it sometimes complains. Will keep this in mind but still little hard to follow this colors since they are not directly referenced in design.